process_report/tests/unit_tests.py

```diff
@@ -160,155 +190,225 @@ def test_export_pi(self):
-        self.assertEqual(pi_df['Manager (PI)'].unique()[0], self.dataframe['Manager (PI)'][0])
+        pi_df = pandas.read_csv(output_dir.name + "/" + pi_csv_1)
+        self.assertEqual(len(pi_df["Manager (PI)"].unique()), 1)
+        self.assertEqual(
+            pi_df["Manager (PI)"].unique()[0], self.dataframe["Manager (PI)"][0]
+        )
 
-        self.assertIn('ProjectA', pi_df['Project - Allocation'].tolist())
-        self.assertIn('ProjectB', pi_df['Project - Allocation'].tolist())
-        self.assertIn('ProjectC', pi_df['Project - Allocation'].tolist())
+        self.assertIn("ProjectA", pi_df["Project - Allocation"].tolist())
+        self.assertIn("ProjectB", pi_df["Project - Allocation"].tolist())
+        self.assertIn("ProjectC", pi_df["Project - Allocation"].tolist())
 
-        pi_df = pandas.read_csv(output_dir.name + '/' + pi_csv_2)
-        self.assertEqual(len(pi_df['Manager (PI)'].unique()), 1)
-        self.assertEqual(pi_df['Manager (PI)'].unique()[0], self.dataframe['Manager (PI)'][3])
+        pi_df = pandas.read_csv(output_dir.name + "/" + pi_csv_2)
+        self.assertEqual(len(pi_df["Manager (PI)"].unique()), 1)
+        self.assertEqual(
+            pi_df["Manager (PI)"].unique()[0], self.dataframe["Manager (PI)"][3]
+        )
 
-        self.assertIn('ProjectD', pi_df['Project - Allocation'].tolist())
-        self.assertIn('ProjectE', pi_df['Project - Allocation'].tolist())
-        self.assertNotIn('ProjectA', pi_df['Project - Allocation'].tolist())
-        self.assertNotIn('ProjectB', pi_df['Project - Allocation'].tolist())
-        self.assertNotIn('ProjectC', pi_df['Project - Allocation'].tolist())
+        self.assertIn("ProjectD", pi_df["Project - Allocation"].tolist())
+        self.assertIn("ProjectE", pi_df["Project - Allocation"].tolist())
+        self.assertNotIn("ProjectA", pi_df["Project - Allocation"].tolist())
+        self.assertNotIn("ProjectB", pi_df["Project - Allocation"].tolist())
+        self.assertNotIn("ProjectC", pi_df["Project - Allocation"].tolist())
 
 
 class TestGetInstitute(TestCase):
     def test_get_pi_institution(self):
-
         institute_map = {
-            "harvard.edu"           : "Harvard University",
-            "bu.edu"                : "Boston University",
-            "bentley.edu"           : "Bentley",
-            "mclean.harvard.edu"    : "McLean Hospital",
-            "meei.harvard.edu"      : "Massachusetts Eye & Ear",
-            "dfci.harvard.edu"      : "Dana-Farber Cancer Institute",
-            "northeastern.edu"      : "Northeastern University",
+            "harvard.edu": "Harvard University",
+            "bu.edu": "Boston University",
+            "bentley.edu": "Bentley",
+            "mclean.harvard.edu": "McLean Hospital",
+            "meei.harvard.edu": "Massachusetts Eye & Ear",
+            "dfci.harvard.edu": "Dana-Farber Cancer Institute",
+            "northeastern.edu": "Northeastern University",
```

Comment:
I liked the original formatting because it was nice to look at, but it's okay to stick to whatever the linter is saying lol.